### PR TITLE
Make tests pass with plain prove

### PR DESCRIPTION
**Pull request by HaraldJoerg**
When I run the tests from the repository with `prove -vlbr`, then two tests (`basic.t `and `test_control_methods`) have one failure each.  The failure disappears when I directly call `prove -vlbr t/basic.t`.  Apparently `prove` without a path prepends `./t/`  before the test files: the failure reappears when I call `prove -vlbr ./t/basic.t`.  The patch allows for an optional `./` before the path names in the `die` messages.
`make test` passes with and without the patch.

This PR is a "side effect" of my work with test-class-moose as my July assignment in the CPAN Pull Request Challenge.

### CODE REVIEW

File: t/basic.t [hunk 1]
```diff
@@ -227,7 +227,7 @@ subtest_streamed(
                             call subevents => array {
                                 event Exception => sub {
                                     call error => match
-                                      qr{\Qforced die at t\E.\Qbasic.t\E.+}s;
+                                      qr{\Qforced die at \E(\./)?\Qt\E.\Qbasic.t\E.+}s;
```
Comment (autarch) [inline t/basic.t]: The parens are unnecessary and I'm not sure this is the right fix anyway. I think it'd be better to rewrite the regex as something like this `qr{\Qforced die at \E.*\Qbasic.t\E.+}s`

We don't really care much about the path leading up to the filename, we just want to make sure it's reporting the right exception.

File: t/test_control_methods.t [hunk 2]
```diff
@@ -60,7 +60,7 @@ subtest_streamed(
                         };
                         event Diag => sub {
                             call message => match
-                              qr/\Qforced die at t\E.\Qtest_control_methods.t\E.+/s;
+                              qr/\Qforced die at \E(\.\/)?\Qt\E.\Qtest_control_methods.t\E.+/s;
```
Comment (autarch) [inline t/test_control_methods.t]: Same applies here.

**Comment by HaraldJoerg** [comment]
Agreed, and adjusted accordingly.

**Comment by autarch** [comment]
Merged from the CLI. Thanks!

**Pull request closed without merging**